### PR TITLE
Fix RuboCop tests.

### PR DESCRIPTION
**Pull request by reitermarkus**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/homebrew/pull/49031).
- [x] Have you successfully run `brew tests` with your changes locally?

### CODE REVIEW

**Comment by MikeMcQuaid** [comment]
Can you explain how/why they are broken and need "fix"ed? Not disagreeing, just want to understand. CC @GauthamGoli too.

**Comment by MikeMcQuaid** [comment]
> Can you explain how/why they are broken and need "fix"ed?

(ideally also in the commit message)

**Comment by reitermarkus** [comment]
The tests are still running with RuboCop 0.49 because of the `Gemfile.lock`, and some methods have changed from 0.49 to 0.50.

File: Library/Homebrew/test/Gemfile [hunk 1]
```diff
@@ -1,10 +1,12 @@
 source "https://rubygems.org"
 
+require_relative "../constants"
+
 gem "parallel_tests"
 gem "rspec"
 gem "rspec-its", require: false
 gem "rspec-wait", require: false
-gem "rubocop"
+gem "rubocop", HOMEBREW_RUBOCOP_VERSION
```
Comment (MikeMcQuaid) [inline Library/Homebrew/test/Gemfile]: 👍

**Comment by GauthamGoli** [comment]
Ah! This fixes RuboCop in Homebrew now. I have been looking at the wrong place. 
Thanks @reitermarkus

**Comment by ilovezfs** [comment]
It looks like this addresses https://github.com/Homebrew/brew/issues/3286. I'd like to get this merged ASAP since we're currently without functioning brew audit.

**Comment by ilovezfs** [comment]
Merged. Hopefully nothing blows up but if it does, it's my fault not @reitermarkus's :)

**Pull request merged**